homeassistant/config_entries.py

```diff
@@ -1133,6 +1133,14 @@ def async_setup_platforms(
         for platform in platforms:
             self.hass.async_create_task(self.async_forward_entry_setup(entry, platform))
```

Comment:
Should we add a deprecation warning to `async_setup_platforms`?

Reply:
a7955863f9ba6b81a7df5c1aa8a28d4dd4147169